util/topsql/collector/mock/mock.go

```diff
@@ -118,6 +118,20 @@ func (c *TopSQLCollector) GetSQLStatsBySQL(sql string, planIsNotNull bool) []*co
 	return stats
 }
 
+// GetSQLCPUTimeBySQL uses for testing.
+func (c *TopSQLCollector) GetSQLCPUTimeBySQL(sql string) uint32 {
```

Comment:
This is a mock collector. How about CPU time in real life?

Reply:
The CPU time of SQL is real data, not a mock data.